runtime/lua/vim/_defaults.lua

```diff
@@ -200,6 +200,171 @@ do
       desc = 'Open a floating window showing diagnostics under the cursor',
     })
   end
+
+  --- vim-unimpaired style mappings. See: https://github.com/tpope/vim-unimpaired
+  do
+    -- Quickfix mappings
+    vim.keymap.set('n', '[q', function()
+      vim.cmd.cprevious({ count = vim.v.count1 })
+    end, {
+      desc = ':cprevious',
+    })
+
+    vim.keymap.set('n', ']q', function()
+      vim.cmd.cnext({ count = vim.v.count1 })
+    end, {
+      desc = ':cnext',
+    })
+
+    vim.keymap.set('n', '[Q', function()
+      vim.cmd.crewind({ args = vim.v.count ~= 0 and { vim.v.count } or nil })
+    end, {
+      desc = ':crewind',
+    })
+
+    vim.keymap.set('n', ']Q', function()
+      vim.cmd.clast({ args = vim.v.count ~= 0 and { vim.v.count } or nil })
+    end, {
+      desc = ':clast',
+    })
+
+    vim.keymap.set('n', '[<C-Q>', function()
+      vim.cmd.cpfile({ count = vim.v.count1 })
+    end, {
+      desc = ':cpfile',
+    })
+
+    vim.keymap.set('n', ']<C-Q>', function()
+      vim.cmd.cnfile({ count = vim.v.count1 })
+    end, {
+      desc = ':cnfile',
+    })
+
+    -- Location list mappings
+    vim.keymap.set('n', '[l', function()
+      vim.cmd.lprevious({ count = vim.v.count1 })
+    end, {
+      desc = ':lprevious',
+    })
+
+    vim.keymap.set('n', ']l', function()
+      vim.cmd.lnext({ count = vim.v.count1 })
+    end, {
+      desc = ':lnext',
+    })
+
+    vim.keymap.set('n', '[L', function()
+      vim.cmd.lrewind({ args = vim.v.count ~= 0 and { vim.v.count } or nil })
+    end, {
+      desc = ':lrewind',
+    })
+
+    vim.keymap.set('n', ']L', function()
+      vim.cmd.llast({ args = vim.v.count ~= 0 and { vim.v.count } or nil })
+    end, {
+      desc = ':llast',
+    })
+
+    vim.keymap.set('n', '[<C-L>', function()
+      vim.cmd.lpfile({ count = vim.v.count1 })
+    end, {
+      desc = ':lpfile',
+    })
+
+    vim.keymap.set('n', ']<C-L>', function()
+      vim.cmd.lnfile({ count = vim.v.count1 })
+    end, {
+      desc = ':lnfile',
+    })
+
+    -- Argument list
```

Comment:
I never understood the use of the argument list. Why would one use this over `bnext` and `bprev`?